01-Getting-Started.md

```diff
@@ -299,6 +301,14 @@ Assume POSIX functions aren't available on Windows, and prefer Boost or C++ stan
 
 TODO: Add ways to get common paths on Windows.
 
+### Internationalization (i18n)
+
+In light of today's global marketplace for software, internationalization support (abbreviated "i18n") is critical, and Puppet is making strides towards full i18n-compliance across all products. At a minimum, compliant projects must address translation of user-visible strings and provide correct handling of Unicode (UTF-8).
+
+#### Extracting and Translating Text
+
+Leatherman provides support for marking translatable strings and generating and using message catalogs (.pot files) based on the `gettext` library. Please refer to the [`Leatherman documentation`](https://github.com/puppetlabs/leatherman#internationalization-i18n) for details.
```

Comment:
I'd change translatable to localizable (as it may include translation, but also changing formatting for specific locales).